src/main/java/org/sonarsource/scanner/cli/Cli.java

```diff
@@ -84,6 +89,9 @@ private int processNextArg(String[] args, int pos) {
     } else if (asList("-D", "--define").contains(arg)) {
       return processProp(args, pos);
 
+    } else if ("--embedded".equals(arg)) {
```

Comment:
I re-used the name suggested in the ticket, feel free to suggest something else.